takwimu/urls.py

```diff
@@ -34,6 +35,7 @@
     url(r'^sdgs/$', SDGIndicatorView.as_view(), name='sdgs'),
     url(r'^feed/$', CountryProfileFeed(), name='rss_feed'),
     url(r'^search/$', SearchView.as_view(), name='search'),
+    url(r'^api/search/$', SearchAPIView.as_view(), name='api-search'),
```

Comment:
Just a comment, this doesn't affect Wagtail/Wazimap API at all, right?

Reply:
@kilemensi Wagtail API is at `api/v2`